picard/ui/metadatabox.py

```diff
@@ -132,21 +136,24 @@ def __tag_ne(self, tag, orig, new):
         else:
             return orig != new
 
-    def add(self, tag, orig_values, new_values, removable, removed=False, readonly=False):
+    def add(self, tag, orig_values, new_values, removable, removed=False, readonly=False, top_tags=None):
         if orig_values:
             self.orig.add(tag, orig_values)
 
         if new_values:
             self.new.add(tag, new_values)
 
+        if not top_tags:
+            top_tags = []
```

Comment:
Instead of a list, since it is only used below as `tag in top_tags`, and never modified, we could use (and pass a set).